content/22.vaccines.md

```diff
@@ -171,7 +171,66 @@ As of December 11, 2020, the United States FDA approved this vaccine under an em
+The rAd26-S is administered first, followed by rAd5-S 21 days later.
+Both vaccines deliver 10^11 viral particles per dose. 
+This approach is designed to overcome any potential pre-existing immunity to adenovirus in the population [@doi:10.1016/S0140-6736(21)00191-4] as some individuals may possess immunity to Ad5 [@doi:10.1016/j.vaccine.2011.05.025].
+Sputnik V is the only recombinant adenovirus vaccine to utilize two vectors, unlike other vaccines such as the Oxford-AstraZeneca vaccine, which utilizes the chimpanzee adenovirus vector (ChAdOx1 nCoV-19) for both doses [@doi:ghpghz].
+The Sputnik V vaccines are available in both a lyophilized (Gam-COVID-Vac-Lyo) and frozen form (Gan-COVID-Vac), which are stored at 2-8 °C and -18 °C respectively [@doi:gg96hq]. 
+The lyophilized vaccine is convenient for distribution and storage, particularly to remote or disadvantaged areas [@doi:10:3390/v13010054].
+
+In the race to develop vaccines against SARS-CoV-2, President Vladimir Putin of Russia announced the approval of the Sputnik V vaccines on August 11th 2020 in the absence of clinical evidence [@doi:ft7j]. 
+Consequently, many international scientific agencies and public health bodies expressed concern because due diligence to the clinical trial process was subverted for the sake of expediency leading many to question the safety and efficacy of Sputnik V [@doi:10.1126/science.abe3147; @url:https://www.cnbc.com/2020/08/11/scientists-worry-whether-russias-sputnik-v-coronavirus-vaccine-is-safe-and-effective.html; @doi:ft7j].
+Despite regulatory, safety, and efficacy concerns, pre-orders for 1 billion doses of the Sputnik V were reported within days of the vaccine's approval in Russia [@doi:ft7j].
+Almost a month later, the phase I/II trial data was published [@doi:gg96hq].
+
+In the phase I/II trial study conducted between late June and early August 2020, 76 participants (18-60 years old) were split into two groups of 38 participants, which were non-randomised in two hospitals in Russia. 
+In phase I, 9 patients received rAd26 and 9 patients received rAd5-S to assess safety over 28 days. 
+In phase II, at least 5 days post the completion of phase I, 20 patients received a prime-boost vaccination of rAd26-S on day 0 and rAd5-S on day 2, which was administered intramuscularly.
+The phase I/II trial reported that both vaccines were deemed safe and well tolerated. 
+The most common adverse events reported were mild with pain at the injection site (58%), hypothermia (50%), headaches (42%), fatigue (28%), and joint and muscle pain (24%). 
+Seroconversion was observed in all participants three weeks post the second vaccination (day 42) and all participants produced antibodies to the SARS-CoV-2 glycoprotein. 
+Receptor binding domain-specific IgG were high (14,703 and 11,143 respectively), indicating a sufficient immune response from both the frozen and lyophilized versions of the vaccine. 
```

Comment:
```suggestion
RBD-specific IgG levels were high in both the frozen and lyophilized versions of the vaccine (14,703 and 11,143 respectively), indicating a sufficient immune response to both.
```
The RBD acronym is used a few other places in the manuscript, so we should be able to use it here too!